cli/src/lib/npm/npmProjectUtils.js

```diff
@@ -90,6 +97,18 @@ export function getPackageJsonDependencies(
         if (deps[pkgName]) {
           console.warn(`Found ${pkgName} listed twice in package.json!`);
         }
+        const pkgIgnored = ignoreDefs.reduce((acc, cur) => {
```

Comment:
I think you could use `.some` instead of `.reduce` and it'll both fail early and read more easily.